lenses-cloudwatch-plugin/src/main/scala/io/lenses/alerts/plugin/cloudwatch/CloudWatchAlertsPlugin.scala

```diff
@@ -19,13 +21,19 @@ class CloudWatchAlertsPlugin extends AlertingPlugin with Metadata {
   override def init(config: util.Map[String, String]): JTry[AlertingService] = Try {
     val map = config.asScala
 
-    def getOrError(key: String): String = {
+    def getOrError(key: String): String =
       map.getOrElse(key, throw new IllegalArgumentException(s"Invalid configuration for CloudWatch plugin'[$key]'"))
+
+    val authMode = getOrError(AUTH_MODE_KEY)
+    val (accessKey, accessSecretKey) = authMode match {
```

Comment:
Hmm I will change this. For Lenses it does not need to be case sensitive since this value is coming from an enum, but since this is not only for Lenses and a user can perfectly send any property, It's indeed a good idea to make it case-insensitive. Thanks for spotting!